### ISSUE TITLE
Stat Tracking past 5 minutes impossible

### ISSUE THREAD

**Issue by 733737** [OpenRCT2/OpenRCT2#16277]
### Operating System

Windows 11 Beta Insider

### OpenRCT2 build

OpenRCT2, v0.3.5.1 (bd9f7d5 on develop) provided by GitHub

### Describe the issue

When designing a ride with a time length of more than 5 minutes the stat tracker cuts off the rest of the ride. It is difficult to fix the stats on my coaster.

### Steps to reproduce

1. Create long ride
2. click on stat tracker
3. All stats cut off at 5 minutes

### Attachments

![733737 Giga Park 2021-12-19 12-32-34](https://user-images.githubusercontent.com/94884086/146684853-d6549ec5-1bbe-4183-b7a9-e41d7e1a600c.png)

**Comment by duncanspumpkin** [comment]
Yes there is a limit to how much data can be recorded you have reached the limit

**Comment by 733737** [comment]
Is it too difficult to increase it?

**Comment by duncanspumpkin** [comment]
Hmm not sure. Suspect a few things would need changed.

**Comment by TheClaudeQC** [comment]
that also exist in rct2, I rememeber that I made very long rides and that limit was reached easily
